assets/scripts/date.js

```diff
@@ -10,7 +10,7 @@ function doPopupDate(qId) {
         var format=$(this).data("DateTimePicker").format();
         /* unsure of this part : don't need format ? */
         minDate = moment(minDate.substr(0,10), "YYYY-MM-DD");
-        maxDate = moment(maxDate.substr(0,10), "YYYY-MM-DD");
+        maxDate = moment(maxDate.substr(0,10) + " " + "23:59:59.999", "YYYY-MM-DD hh:mm:ss.SSS");
```

Comment:
What happen if format is `d/m/yy HH:MM` and (for example) max date is "+1 hour" ?

Reply:
OK : broken (currently)